### ISSUE TITLE
Introduce "run" command

### ISSUE THREAD

**Issue by gr2m** [octoherd/cli#23]
Instead of 

```
octoherd.js [script] [repos...]
```

I want to use

```
octoherd run [script] [repos...]
```

There will be other commands in future that I would like to introduce, such as `login`, `whoami`, `logout`, and `create`.

**Comment by oscard0m** [comment]
Nice!

---

It reminds me to the discussion on npm regarding `npm run <script>` vs `npm <script>`, what do you think about `octoherd [script]` instead of `octoherd run [script]`?

**Comment by gr2m** [comment]
`npm` and `octoherd` are very different CLIs. I'd go with `octoherd run` for now, it's more clear. And we already export a `run` method that we use in the script repositories:

https://github.com/octoherd/script-hello-world/blob/a86b8d629bd4f1a38563cc758ae449564b843579/cli.js

Using the `octoherd run` command explicitly makes things more consistent. But it's not a decision we couldn't reverse in future

**Comment by oscard0m** [comment]
> `npm` and `octoherd` are very different CLIs. I'd go with `octoherd run` for now, it's more clear. And we already export a `run` method that we use in the script repositories:
> 
> https://github.com/octoherd/script-hello-world/blob/a86b8d629bd4f1a38563cc758ae449564b843579/cli.js
> 
> Using the `octoherd run` command explicitly makes things more consistent. But it's not a decision we couldn't reverse in future

Sure. Let's go for it :)

**Comment by gr2m** [comment]
released in v3.1
